packages/teams-js/src/public/app.ts

```diff
@@ -732,10 +742,12 @@ function transformLegacyContextToAppContext(legacyContext: LegacyContext): app.C
     sharePointSite:
       legacyContext.teamSiteUrl || legacyContext.teamSiteDomain || legacyContext.teamSitePath
```

Comment:
should this list include the new properties too?

Reply:
as in, is it possible to have `mySiteUrl` present on context with all these being undefined?

Reply:
@MarcinJuraszek Yes. I've made this update.